lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -77,7 +79,7 @@ class StartUrl extends Gatherer {
 
   afterPass(options, tracingData) {
     if (!this.startUrl) {
-      return Promise.reject(new Error('No start_url found inside the manifest'));
+      return Promise.reject(ExpectedError(new Error('No start_url found inside the manifest')));
```

Comment:
if we go with `ExpectedError`, it should definitely be `new ExpectedError` or `createExpectedError` and should settle on whether to pass a string or an exception in